src/layouts/Catalogue/Catalogue.js

```diff
@@ -187,58 +188,113 @@ export default class Catalogue extends Component {
           <img src={this.state.selectedProduct.image}
               style={{ height: "250px" }}></img>
           <div style={{ margin: "20px", padding: "10px" }}>
-            <div><b>{this.state.selectedProduct.title}</b></div>
-            <div style={{ fontSize: "12px" }}>
-              {this.state.selectedProduct.description}<br/>
-              <b>${this.state.selectedProduct.price}</b><br/>
-            </div>
-            <div>
-              <InputNumber min={1} defaultValue={1} onChange={(value) => {console.log("Number Value :",value); this.onChangeQty(value)}}/>
-            </div>
+            <h2>{this.state.selectedProduct.title}</h2>
+            <p style={{ fontSize: "12px", display: "block" }}>
+              {this.state.selectedProduct.description}
+              <strong>${this.state.selectedProduct.price}</strong>
+            </p>
+            <InputNumber min={1} defaultValue={1} onChange={(value) => {console.log("Number Value :",value); this.onChangeQty(value)}}/>
           </div>
         </div>
         
       </Modal>
     )
   }
 
-  renderNavBar() {
-    const { navigate } = this.props;
+  renderNavBar() {    
+    const NavBar = styled.nav`
+      width: 100vw;
+      height: 70px;
+      display: flex;
+      position: fixed;
+      top: 0;
+      padding: 0;
+      z-index: 3;
+      justify-content: center;
+      background: #1c4d86;
+      align-items: center;
+    `; 
+
+    const Container = styled.div`
+      color: white;
+      width: 85%;
+      margin-left: 0;
+      display: flex;
+      justify-content: space-between;
+    `;
+
+    const Left = styled.div`
+      display: flex;
+      margin: 0;
+      flex: 2;
+      justify-content: flex-start;
+    `;
+
+    const Right = styled.div`
+      display: flex;
+      flex: 2;
+      justify-content: flex-end;
+    `;
+
     return (
-      <section className="navbar">
-        <div className="navbar-container">
-          <div className="navbar-left">
-            <ul className={this.state.menuOpened ? "active" : "none"}>
-              <li
-                onClick={() => {
-                  navigate("/");
-                  this.handleMenuOpen();
-                }}
-              >
+      <NavBar>
+        <Container>
+          <Left>
```

Comment:
I think Left and Right are a bit too common. I'd suggest LeftColumn and RightColumn.